### DISCUSSION TITLE
Very slow downloads for users from my Nextcloud server. Server has 10G bandwidth, and both Speedtest and Iperf confirm fast connections.

### DISCUSSION THREAD

**Discussion by HydrogenHuman** [nextcloud/all-in-one#4484] [Questions] [answered]
I've been trying to figure out a way to improve the download speeds for users when they download files from my Nextcloud server. Currently users, including myself, experience download speeds of about 1Mbps. Sometimes when the download initially starts, it can be up to around 3Mbps, but quickly drops to 1Mbps.

Note: Uploading is also just as slow.

I have performed Iperf and Speedtest CLI tests on numerous servers from around the globe, and they all consistently show a much faster bandwidth than what I'm getting from Nextcloud itself.

I have been unable to find any answer to this problem through google searches, and it seems to be a common problem. 

I have the latest AIO update as well.

**Comment by szaimen** [comment, marked as answer]
This will be handled in https://help.nextcloud.com/t/nextcloud-aio-server-ubuntu-22-04-has-very-slow-download-speeds-for-the-end-user/187019
